packages/css/src/components/form-error-list/form-error-list.scss

```diff
@@ -0,0 +1,4 @@
+.ams-form-error-list {
+  // The offset is doubled to compensate for the Alert border
+  outline-offset: var(--ams-form-error-list-outline-offset);
```

Comment:
Why do we need to double the offset if “its line is outside of the element's border” ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/outline#description))?

Reply:
Additional research indicated we need to add an `outline-width: medium` reset: Safari on Mac seems to have an abnormal initial value of `thick` here, which explains the extra whitespace between border and outline.

Reply:
I’ve added this line to this component for now. We’ll research this further and expand to other components if necessary, in a new ticket & PR.